bindings/pydeck/pydeck/bindings/layer.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, type, data, id=None, use_binary_transport=None, **kwargs):
         here will be specific to the particular deck.gl layer that you are choosing to use.
 
         Please see the deck.gl
-        `Layer catalog <https://deck.gl/#/documentation/deckgl-api-reference/layers/overview>`_
+        `Layer catalog <https://deck.gl/docs/api-reference/layers>`_
```

Comment:
`/docs/api-reference/layers/README.md`

Reply:
The link I put will also link to the README

Reply:
The local link format will lead to the actual file when you click it on GitHub, and get dynamically replaced when the website is compiled.

Reply:
This is more desirable than linking directly to website because if someone browses the documentation from a past release branch, link to website may be dead and/or does not reflect the state of that particular release.

Reply:
I think that's only for markdown files. This is an RST. Can you confirm that syntax will work for RST?
https://docutils.sourceforge.io/docs/user/rst/quickref.html

Side note: While checking other RST files in this library for links I found that many heading links are broken https://github.com/visgl/deck.gl/blob/7c88277de1357679f2b94d196f860d9dc161d6ce/bindings/pydeck/docs/index.rst . Will work on those later once I learn how RST link syntax work!

Reply:
Ah yes you are right. The link remapping only work in markdowns.

Reply:
I'll look up how to do the equivalent in RST tonight.